ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Carrier.java

```diff
@@ -79,13 +97,19 @@ public Spatial getDefaultModel() {
         return ProjectAssetManager.getInstance().getAssetManager().loadModel(PATHNAME);
     }
 
-
     /**
      * Get relativeLocation attribute.
      * @return relativeLocation attribute
      */
     public Vector3f getRelativeLocation() {
         return relativeLocation;
     }
-
+    
+    /**
+     * Getter for healthMessenger.
+     * @return healthMessenger HealthMessenger
```

Comment:
This problem is many more other places in your code:
The return description is very weird. `@return <description>` is how it's supposed to look like. And the <description> is a simple description of what is being returned.
`healthMessenger HealthMessenger` is not really a description. It's just what you're returning and it's type, and the type is very redundant as it's automatically specified in generated JavaDoc.